airflow/migrations/versions/0032_3_0_0_rename_execution_date_to_logical_date_and_nullable.py

```diff
@@ -17,9 +17,9 @@
 # under the License.
 
 """
-Drop ``execution_date`` unique constraint on DagRun.
+Make logical_date nullable.
```

Comment:
Editing a migration will not re-run anything for people using breeze. (They might run into silent errors until they clean their db and re-run migration from scratch). 

That's not a big deal but we might want to drop a message in the contributor channel.